scripts/install.go

```diff
@@ -0,0 +1,62 @@
+package scripts
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+//go:embed setup/setup.sh
+//go:embed setup/config-template.toml
+//go:embed setup/coriolis-snapshot-agent.service.sample
+var setupContent embed.FS
+
+const (
+	setupTmpDir = "/tmp/setup"
+)
+
+func writeEmbeddedFile(filename string, mode fs.FileMode) error {
+	fileContent, err := setupContent.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+
+	return os.WriteFile(filepath.Join("/tmp", filename), fileContent, mode)
+}
+
+func RunInstall() {
+	if err := os.MkdirAll(setupTmpDir, 0700); err != nil {
+		log.Fatal(err)
+	}
+	defer os.RemoveAll(setupTmpDir)
+
+	if err := writeEmbeddedFile("setup/setup.sh", 0700); err != nil {
```

Comment:
This would need to get the temp dir created above:

```suggestion
	if err := writeEmbeddedFile("setup/setup.sh", dir, 0700); err != nil {
```